### ISSUE TITLE
Throwing HubException on server and getting the error message on typescript client

### ISSUE THREAD

**Issue by IonutArhire** [aspnet/SignalR#2437]
Here is the back-end method I'm invoking:

```
public async Task SendResources(string gameKey) {
      throw new HubException("oops");
      ...
}
```
In my typescript client I have:

```
public connected(data): void {    
  this._hubConnection.invoke('SendResources', this._gameKey).catch(this.errorHandler);
}

public errorHandler(err): void {
  console.log(err);
}
```
The execution flow enters 'errorHandler' with 'err' being of type Error. The problem is that I can't access the message of the exception. All info I get from the 'err' variable is this:

![capture](https://user-images.githubusercontent.com/26301880/41033716-2a5d3b04-6990-11e8-8def-53f696a2e881.PNG)

I followed what the docs said: https://docs.microsoft.com/en-us/aspnet/core/signalr/hubs?view=aspnetcore-2.1#handle-errors

Any Idea how to handle error messages correctly with signalR?

Versions:
.net core: 2.1
angular: 5
@aspnet/signalr: 1.0.0

**Comment by BrennanConroy** [comment]
Error messages aren't propagated to the client by default as they can have sensitive information. You can set the `EnableDetailedErrors` property to true when setting up SignalR and that will allow the messages through.

I've opened https://github.com/aspnet/SignalR/issues/2439 because we probably will want `HubException` to ignore the `EnableDetailedErrors` flag.

**Comment by anurse** [comment]
I'm going to close this since @BrennanConroy opened a new issue for the actionable work here.

**Comment by mikaelm12** [comment]
This is resolved here c7ebae47ea3569183a27c1edd745d79da7c8f159
